src/lib.rs

```diff
@@ -103,12 +108,20 @@ impl fmt::Debug for Library {
 ///
 /// # Examples
 ///
-/// ```ignore
-/// let sin: Symbol<extern fn(f64) -> f64> = unsafe {
+#[cfg_attr(all(unix, not(any(target_os="macos", target_os="ios", target_os="android"))),
+           doc = " ```")]
+#[cfg_attr(not(all(unix, not(any(target_os="macos", target_os="ios", target_os="android")))),
```

Comment:
Just `no_run` them on all platforms. From the standpoint of maintenance, this seems like an overly expensive workaround for lack of features in rustdoc.